.gitmodules

```diff
@@ -7,3 +7,33 @@
 [submodule "roles/base"]
 	path = roles/base
 	url = https://github.com/n0emis/ansible-role-base
+[submodule "roles/geerlingguy.php"]
+	path = roles/geerlingguy.php
+	url = https://github.com/geerlingguy/ansible-role-php
+[submodule "roles/geerlingguy.nginx"]
```

Comment:
This might be more of a long-term (and aesthetic) question: Is there a big advantage in using both nginx and traefik on the same host? Or one replace the other?

Reply:
In general yes, but we prefer traefik for it's simple configuration (eg. with letsencrypt). But it can't serve static files, therefore we use nginx for that.